src/main/java/com/nicusa/UiApplication.java

```diff
@@ -24,7 +29,50 @@
       model.put("content", "Hello World");
       return model;
     }
-    
+
```

Comment:
Are the web clients ever going to access drug data from our ReST controller.  They could go directly to the OpenFDA datastream from javascript and have full access to the data rather than us wrapping the data.